programs/leap-util/actions/blocklog.hpp

```diff
@@ -31,7 +31,7 @@ class blocklog_actions : public sub_command<blocklog_options> {
    void initialize();
    int trim_blocklog_end(bfs::path block_dir, uint32_t n);
    bool trim_blocklog_front(bfs::path block_dir, uint32_t n);
-   bool extract_block_range(bfs::path block_dir, bfs::path output_dir, uint32_t start, uint32_t end);
+   void extract_block_range(bfs::path block_dir, bfs::path output_dir, uint32_t start, uint32_t end);
```

Comment:
Without context, I would assume `end` was `last+1`. Let me suggest we update the variable to `last` here.
```suggestion
   void extract_block_range(bfs::path block_dir, bfs::path output_dir, uint32_t start, uint32_t last);
```